java/registry/src/main/java/dev/sunbirdrc/registry/controller/RegistryEntityController.java

```diff
@@ -167,6 +168,10 @@ public ResponseEntity<Object> searchEntity(@PathVariable String entityName, @Req
                 responseParams.setStatus(Response.Status.UNSUCCESSFUL);
                 responseParams.setErrmsg(String.format("Searching on entity %s not allowed", entityName));
             }
+        } catch (NullPointerException e) {
```

Comment:
we can throw custom exception when the schema is not found, instead of catch NPE